.github/ISSUE_TEMPLATE/community-submission.yml

```diff
@@ -0,0 +1,70 @@
+name: Community Submission
+description: Submit your community to be included in our listings or initiatives
+title: "[Community Submission] <Community Name>"
+labels: ["submission"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        🙌 Thank you for submitting your community! Please complete the form below with the required information
+  - type: input
+    id: community-name
+    attributes:
+      label: Community Name
+      placeholder: e.g Awesome Dev Asia 😎
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Briefly describe the purpose or focus of your community.
+      placeholder: e.g A group of developers meeting monthly to discuss cloud-native 🧠
+    validations:
+      required: true
+  - type: input
+    id: website
+    attributes:
+      label: Website or Link
+    validations:
+      required: true
+  - type: dropdown
+    id: region
+    attributes:
```

Comment:
This isn't Asia-specific, so we can add other regions like North America, Europe, Africa etc.

Reply:
Noted! I will update soon to add these options 🙂